resources/views/modular/components/button-column.blade.php

```diff
@@ -0,0 +1,8 @@
+<div>
```

Comment:
Small thing, but let's remember to have the params commented at the top of these components to match the others.